builders/manage/dapp/manage-tokens.md

```diff
@@ -0,0 +1,131 @@
+- The account you used when registering the appchain, imported in any of the [supported wallets](/builders/deploy/dapp/#supported-wallets){target=\_blank}
+- Your appchain's Sudo account, also imported in any of the [supported wallets](/builders/deploy/dapp/#supported-wallets){target=\_blank}. 
+
+!!! note
+    The appchain's registration account is always a Substrate one, whereas the appchain's Sudo account depends on the chain type. If the chain is EVM-compatible, then the Sudo account will be an Ethereum type and, otherwise, a Substrate type.
+
+### Retrieving the Registration Account {: #retrieving-registration-account }
+
+If you're unsure what your registration account is, you can query it directly from the Tanssi orchestrator chain, which keeps records of every registered appchain. To do so, head to the **Chain state** section on the Polkadot.js Apps connected to the orchestrator chain for [snap appchains](https://polkadot.js.org/apps/?rpc=wss://fraa-flashbox-2607-rpc.a.stagenet.tanssi.network#/chainstate){target=\_blank} or [dedicated appchains](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffraa-dancebox-rpc.a.dancebox.tanssi.network#/chainstate){target=\_blank}, and take the following steps:
+
+1. Select the **registrar** storage module
+2. Select **registrarDeposit**
+3. Insert your appchain id
+4. Press **+** icon
+
+You'll see the registration account at the bottom.
+
+![Locating your registration account](/images/builders/manage/dapp/token-management/token-management-1.webp)
+
+### Retrieving the Sudo Account {: #retrieving-sudo-account }
+
+If you're unsure what your Tanssi appchain's Sudo account is, you can find it in your [Tanssi Dashboard](https://apps.tanssi.network){target=\_blank} underneath the **Properties** section.
+
+![Locating your Sudo address on apps.tanssi.network](/images/builders/manage/dapp/token-management/token-management-2.webp)
+
+!!! warning
+    You should always protect your Sudo account key with the utmost security precautions, as it grants privileged access to your Tanssi appchain.
+
+## Accessing the Token Management Panel {: #accesing-token-management-panel }
+
+The Tanssi dApp implements a smooth interface, allowing the appchain governor to access and execute privileged functions. To do so, head to the [Tanssi dApp](https://apps.tanssi.network/){target=\_blank}, click on the **Manage** button, and then, again click on the **Token Management** button.
+
+![Accessing the token management panel](/images/builders/manage/dapp/token-management/token-management-3.webp)
+
+Now you have direct access to the actions presented in the introduction:
+
+1. **Mint Tokens**
+2. **Update Balances**
+3. **Transfer Tokens**
+4. **Configure Gas Dynamics** 
+
+![The token management panel](/images/builders/manage/dapp/token-management/token-management-4.webp)
+
+!!! note
+    If you don't see a card with your appchain's details, make sure to comply with the [prerequisites](#checking-prerequisites).
```

Comment:
Do you mean the **Properties** section?

Reply:
No, I meant the prerequisites section.
I know that complying with the prerequisites is a requisite, but the appchain will be visible in the dashboard only if everything is well configured. Otherwise, double-check the requirements.

Reply:
yes, but when you refer to "card" are you referring to the "Properties" card on the dashboard or this section of the dashboard? If so, can you please update the text to tell them what they're looking at

Reply:
Hmm, I see it is ambiguous. Rephrased!